### ISSUE TITLE
Fetch API : Fetch_types namespace

### ISSUE THREAD

**Issue by et1975** [fable-compiler/fable-powerpack#7]
This might be subjective, but according to certain API design guidelines, API namespace should include all the types required to work with it.

Currently I have to include `Fetch_types` to do anything useful with Fetch API, which perhaps shouldn't be necessary?

**Comment by alfonsogarciacaro** [comment]
Yes, you're right. I actually don't remember why it was done like this. I thought once of changing it but I was afraid of breaking existing code. Maybe the transit to 1.0 is a good chance for this. PR? ;)
